nix/shell.nix

```diff
@@ -26,11 +27,14 @@ pkgs.mkShell rec {
       pkgs.sqlite
       pkgs.zlib
       pkgs.zstd
+      pkgs.vulkan-loader
```

Comment:
This should probably be in the `lib.optionals stdenv.hostPlatform.isLinux` section